extra_tests/snippets/stdlib_hashlib.py

```
@@ -56,3 +56,13 @@
 assert _md5.md5(b"").hexdigest() == "d41d8cd98f00b204e9800998ecf8427e"
 assert _sha1.sha1(b"").hexdigest() == "da39a3ee5e6b4b0d3255bfef95601890afd80709"
+
+# a derived key wider than a C int does not fit, and never gets allocated.
+# Which OverflowError comes out depends on the width of a C long: where it is
+# narrower than the length asked for, converting the argument fails first.
+try:
+    hashlib.pbkdf2_hmac("sha256", b"password", b"salt", 1, 2**62)
+except OverflowError:
+    pass
+else:
+    assert False, "expected OverflowError"
```

Ruff (0.16.1)

[warning] Do not `assert False` (`python -O` removes these calls), raise `AssertionError()`

Replace `assert False`

(B011)